python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -3272,6 +3272,21 @@ def test_error_stack_trace(self):
         )
         spark.stop()
 
+    def test_can_create_multiple_sessions_to_different_remotes(self):
```

Comment:
test to validate connecting to different remotes works is missing?

Reply:
Done